### ISSUE TITLE
Is there any way to boot Rapi from SSD installed to Quad SATA Hat?

### ISSUE THREAD

**Issue by lovey25** [akgnah/rockpi-sata#13]
I want to boot RaPi up directly from an SSD with Raspberry Pi OS without using an SD card.
Is there any way to set GPIO25, 26 high, when the Raspberry Pi is booting so that it can boot directly from a SSD connected to SATA Hat?

**Comment by artivis** [comment]
You may want to check [this comment](https://forum.radxa.com/t/not-possible-ssd-boot-from-quad-sata-hat/5045/16) on the forum.

Tl;dr

```bash
$ sudo -E rpi-eeprom-config --edit
```

Add the following at the very end of the file,

```
[config.txt]
gpio=25,26=op,dh
```

Find out more about this from the doc [here](https://www.raspberrypi.org/documentation/configuration/config-txt/gpio.md) and [here](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloader_config.md).

Note: If you haven't enabled booting from USB device, do it before adding those lines otherwise the boot option is added automatically at the end of this file by `raspi-config` and this messes up things.

Note: If you are using the provided SATA software scripts then edit the file `/usr/bin/rockpi-sata/misc.py` as follows,

```python
def disk_turn_off():
    # set_mode(26, GPIO.LOW)
    time.sleep(0.5)
    # set_mode(25, GPIO.LOW)
```
